### PR TITLE
Add pytoniq examples

### PR DESCRIPTION
**Pull request by shibdev**

### CODE REVIEW

File: docs/develop/dapps/cookbook.md [hunk 1]
```diff
@@ -402,6 +425,28 @@ println(jettonWalletAddress.toString(userFriendly = true))
...
+    provider = LiteBalancer.from_mainnet_config(1)
+    await provider.start_up()
+
+    JETTON_MASTER_ADDRESS = "EQBlqsm144Dq6SjbPI4jjZvA1hqTIP3CvHovbIfW_t-SCALE"
+    USER_ADDRESS = "EQAsl59qOy9C2XL5452lGbHU9bI3l4lhRaopeNZ82NRK8nlA"
+
+
+    jetton_wallet = (await provider.run_get_method(address=JETTON_MASTER_ADDRESS, method="get_wallet_address",
+                                                   stack=[begin_cell().store_address(USER_ADDRESS).end_cell().begin_parse()]))[0].load_address()
+    print(f"Jetton wallet address for {USER_ADDRESS}: {jetton_wallet.to_str(1, 1, 1)}")
```
Comment (yungwine) [inline docs/develop/dapps/cookbook.md:444]:
```suggestion
    result_stack = await provider.run_get_method(address=JETTON_MASTER_ADDRESS, method="get_wallet_address",
                                                   stack=[begin_cell().store_address(USER_ADDRESS).end_cell().begin_parse()])
    jetton_wallet = result_stack[0].load_address()
    print(f"Jetton wallet address for {USER_ADDRESS}: {jetton_wallet.to_str(1, 1, 1)}")
```

File: docs/develop/dapps/cookbook.md [hunk 2]
```diff
@@ -402,6 +426,30 @@ println(jettonWalletAddress.toString(userFriendly = true))
...
+
+    JETTON_MASTER_ADDRESS = "EQBlqsm144Dq6SjbPI4jjZvA1hqTIP3CvHovbIfW_t-SCALE"
+    USER_ADDRESS = "EQAsl59qOy9C2XL5452lGbHU9bI3l4lhRaopeNZ82NRK8nlA"
+
+
+    result_stack = await provider.run_get_method(address=JETTON_MASTER_ADDRESS, method="get_wallet_address",
+                                                   stack=[begin_cell().store_address(USER_ADDRESS).end_cell().begin_parse()])
+    jetton_wallet = result_stack[0].load_address()
+    print(f"Jetton wallet address for {USER_ADDRESS}: {jetton_wallet.to_str(1, 1, 1)}")
+	await client.close_all()
```
Comment (yungwine) [inline docs/develop/dapps/cookbook.md:447]:
```suggestion
	await provider.close_all()
```

**Comment by yungwine** [comment]
@reveloper, looks like everything is good now and you can merge it!

@shibdev, thank you!

**Comment by reveloper** [comment]
@shibdev, @yungwine that's great! Thank you for this update!

**Pull request merged**